### ISSUE TITLE
Jest 'setupTestFrameworkScriptFile' Deprecation Warning

### ISSUE THREAD

**Issue by samechikson** [meltedspark/angular-builders#239]
**Is your feature request related to a problem? Please describe.**
Not really a problem, just a warning when running tests

```
● Deprecation Warning:

  Option "setupTestFrameworkScriptFile" was replaced by configuration "setupFilesAfterEnv", which supports multiple paths.

  Please update your configuration.

  Configuration Documentation:
  https://jestjs.io/docs/configuration.html
```

**Describe the solution you'd like**
Change https://github.com/meltedspark/angular-builders/blob/9a7cfa092fa9e220ea30143b50194faf51e095a0/packages/jest/src/jest-config/default-config.ts#L7 
to 
```
setupFilesAfterEnv: [`${__dirname}/setup.js`], 
```

**Comment by meltedspark** [comment]
Thanks for reporting! Fixing this will also fix #193.

**Comment by drakenfly** [comment]
Why is this not merged yet?

**Comment by meltedspark** [comment]
Available in `7.4.0`
